webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
```

Comment:
I think that we should declare each variable using `const` on it's own. I see that we're doing it elsewhere so it's a good idea to stick to one way